### ISSUE TITLE
Resiliency4j -> circuitbreaker default configuration is not taking effective.

### ISSUE THREAD

**Issue by Karthickramk** [resilience4j/resilience4j#937]
Resilience4j version: 1.3.0

Java version:Zulu11.31+10-SA (build 11.0.3+7-LTS)

Hi Team,

Issue#1 
I have the below configuration in my application.yaml file. Here when circuit breaker works fine for the operation1. But the deafault configuration is not overridden for other backend operations. Is it a known issue? 

Issue#2
For resiliency4j retry, there is validation for waitDuration that minimum value should be 100ms. I don't see any option to override with less than 100 ms. Why don't we have this option? 

Application.yaml
```
resilience4j.circuitbreaker:
  backends:
    oepration1:
      slidingWindowSize: 6
      permittedNumberOfCallsInHalfOpenState: 3
      waitDurationInOpenState: 5000
      failureRateThreshold: 20
      recordExceptions:
        - com.walmart.sams.astra.common.exception.AstraIOException
    default:
      slidingWindowSize: 6
      permittedNumberOfCallsInHalfOpenState: 3
      waitDurationInOpenState: 5000
      failureRateThreshold: 20
      recordExceptions:
        - com.walmart.sams.astra.common.exception.AstraIOException
```

**Comment by RobWin** [comment]
Hi,

Issue#1:
your configuration is wrong. If you want to override the default configuration, please use:

```
resilience4j.circuitbreaker:
  configs:
    default:
      slidingWindowSize: 6
      permittedNumberOfCallsInHalfOpenState: 3
      waitDurationInOpenState: 5000
      failureRateThreshold: 20
      recordExceptions:
        - com.walmart.sams.astra.common.exception.AstraIOException
```

Issue#2: This is fixed in v1.4.0. But I don't think that lower watDuration makes much sense, because most of the time atemporary issue is not fixed within this short period of time.

**Comment by Karthickramk** [comment]
Thanks @RobWin  for the quick response. 

For issue#2, you are absolutely right. But I cannot change this configuration in production immediately. So I need to retain this value. I see the latest version 1.3.1 in the repo. Is 1.4.0 is going to be released?
